cluster/hook-template.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: v1
 parameters:
 - name: REGISTRY
   required: true
-  value: gcr.io
+  value: docker.io
 - name: DOCKER_REPO
   required: true
-  value: k8s-prow
+  value: arquillian
 - name: VERSION
-  value: v20180403-2f3ad698f
+  value: latest
```

Comment:
We don't publish "latest" so that default won't work. But we always pass it to the template anyway `-p VERSION=$(PROW_VERSION)` so I think we can just remove the default value

Reply:
the same applies also to `ike-prow-template.yaml` right? Since this is the place I took it from :-)

Reply:
Yeah, that's correct. We want to use exact build tag.

Reply:
673ecf5